src/Components/RecentJobs/index.jsx

```diff
@@ -0,0 +1,18 @@
+import RecentBox from "./RecentBox"
+
+function RecentJobs() {
+    
+
+    return (
+        <div className='bg-white flex flex-col px-10 md:px-20'>
```

Comment:
Can you get this whole RecentJobs section to be the same width as the designs, centered on the page. At the moment it's all full width